chromium_src/chrome/browser/ui/views/tabs/tab.cc

```diff
@@ -9,6 +9,10 @@
 #include "brave/browser/ui/views/tabs/vertical_tab_utils.h"
 #include "chrome/browser/ui/layout_constants.h"
 
+#define BRAVE_UI_VIEWS_TABS_TAB_LAYOUT_ADJUST_ICON_POSITION \
+  favicon_bounds.set_y(contents_rect.y() +                  \
```

Comment:
what is this doing exactly? Please comment on why we need set_y here when upstream does not

Reply:
cc: @zenparsing

Reply:
This one definitely needs a comment.

> Upstream is no longer centering the tab favicon vertically within the tab view (likely due to the fact that their tab contents rect happens to be the same height as the favicon). Ensure that the favicon is centered vertically within the tab.

Reply:
Comment added in 29c1410c35ed1b297e4dad8289bb21789b3ed472